staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -554,17 +554,17 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			versionedObject = defaultVersionedObject
 		}
 		reqScope.Namer = action.Namer
-		namespaced := ""
+		namespaced := "Cluster"
```

Comment:
This just affects swagger, it shouldn't be part of the metrics change.  We already have a constant and type for "rest scoping", so we shouldn't be keying off a string here for the metrics instrumentation, we should be keying off the value of apiResource.Namespaced and then passing that to metrics, leaving the swagger alone